tests/rest/key/v2/test_remote_key_resource.py

```diff
@@ -51,10 +56,15 @@ def expect_outgoing_key_request(
         Tell the mock http client to expect an outgoing GET request for the given key
         """
 
-        async def get_json(destination, path, ignore_backoff=False, **kwargs):
+        async def get_json(
+            destination: str,
+            path: str,
+            ignore_backoff: bool = False,
+            **kwargs: Any,
+        ) -> Union[JsonDict, list]:
             self.assertTrue(ignore_backoff)
             self.assertEqual(destination, server_name)
-            key_id = "%s:%s" % (signing_key.alg, signing_key.version)
+            key_id = "%s:%s" % (signing_key.alg, signing_key.version)  # type: ignore[attr-defined]
```

Comment:
Argh. This is one of my bugbears. Does marking `signing_key: signedjson.types.SigningKey` make these errors go away?

Reply:
Perfect. It works.